api/PushManager.json

```diff
@@ -10,20 +10,9 @@
           "chrome_android": {
             "version_added": "42"
           },
-          "edge": [
-            {
-              "version_added": "16",
-              "flags": [
```

Comment:
The flagged entry was first here, so this will improve what people see on MDN by default, yay 😃